File: Basic Coding Problems/02 If - Elses/019 Write a program to find the maximum between two numbers/Java/maximum_between_two_numbers.java
import java.util.Scanner;

public class maximum_between_two_numbers {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter two numbers: ");
        int firstNumber = scanner.nextInt();
        int secondNumber = scanner.nextInt();

        int maxNumber;

        if (firstNumber > secondNumber) {
            maxNumber = firstNumber;
        } else {
            maxNumber = secondNumber;
        }

        System.out.println("The maximum number is: " + maxNumber);
    }
}

File: Basic Coding Problems/02 If - Elses/019 Write a program to find the maximum between two numbers/C++/maximum_between_two_numbers.cpp
#include <iostream>

using namespace std;

int main() {
    int first_number, second_number, max_number;

    cout << "Enter two numbers: ";
    cin >> first_number >> second_number;

    if (first_number > second_number) {
        max_number = first_number;
    } else {
        max_number = second_number;
    }

    cout << max_number << " is the maximum number.\n";

    return 0;
}
